api/src/opentrons/protocol_runner/protocol_runner.py

```
@@ -156,11 +156,16 @@ async def run(
     def _load_json(self, protocol_source: ProtocolSource) -> None:
         protocol = self._json_file_reader.read(protocol_source)
         commands = self._json_translator.translate_commands(protocol)
-        liquids = self._json_translator.translate_liquids(protocol)
-        for liquid in liquids:
-            self._protocol_engine.add_liquid(liquid=liquid)
+        if feature_flags.enable_load_liquid():
+            liquids = self._json_translator.translate_liquids(protocol)
+            for liquid in liquids:
+                self._protocol_engine.add_liquid(liquid=liquid)
         for command in commands:
-            self._protocol_engine.add_command(request=command)
+            if feature_flags.enable_load_liquid() or (
+                not feature_flags.enable_load_liquid()
+                and command.commandType != "loadLiquid"
+            ):
+                self._protocol_engine.add_command(request=command)
```

Instead of silently ignoring `loadLiquid` commands when `enable_load_liquid()` is `False`, should we raise an "unrecognized command" error?